.travis.yml

```diff
@@ -18,7 +18,9 @@ install:
   - pip install -r requirements.txt
 
 script:
-  - zope-testrunner --path . -s tests --color
+  - VERSION=$(curl -s https://crate.io/versions.json | grep crate_testing | tr -d '" ' | cut -d ":" -f2)
+  - ./update.py --cratedb-version ${VERSION} --platform "x64_linux" > amd64/crate/Dockerfile
+  - PATH_TO_IMAGE=amd64/crate zope-testrunner --path . -s tests --color
```

Comment:
I am failing to move it to the github actions. Maybe we would be to rethink how the tests are written and even move them to the official-images tests, see [official-images/test/tests](https://github.com/docker-library/official-images/tree/master/test/tests)? I'd probably follow up on it later.

Reply:
> and even move them to the official-images tests

That would also make it more difficult to adjust the test setup as Dockerfile and tests for it would be separated. I think we should keep them here.